### ISSUE TITLE
Update a11y for in-app web view and profile pages 

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#17130]
_NOTE: This will resolve tickets(bugs) #15053_

## Dev Task 
1. Update the a11y for tickets:
     - #14164
     - #13890
     - #13896
     - #13902
     - #13907

## Designs
- [In-App Web View Example](https://adhoc.invisionapp.com/share/YUYRT9BAPSB#/screens/432493111)
- [Personal & Contact Information Page](https://adhoc.invisionapp.com/share/YUYRT9BAPSB#/screens/431636868)
- [Military Information Page](https://adhoc.invisionapp.com/share/YUYRT9BAPSB#/screens/431636873)
- [Direct Deposit Page](https://adhoc.invisionapp.com/share/YUYRT9BAPSB#/screens/432050367)
- [Edit Direct Deposit Page](https://adhoc.invisionapp.com/share/YUYRT9BAPSB#/screens/431636876)

## Accessibility Acceptance Criteria
1. Everything on the page should be read by the screen reader.
     - If anything is being conveyed visually only (buttons, menu items, list, tables, images, headings, forms, etc) the component type needs to be announced by the screenreader. (Include name (title), role, value)
2. Each page should support text resizing based on the OS font scaling settings without loss of content, content overlap, or horizontal scroll.
3. Each touch target must be at minimum 44px by 44px
4. The app needs to support operability by tap (no gestures) or voice control only. In cases where gestures occur, e.g. scroll, ensure these simple gestures are operable through the OS’ built-in capabilities.  
5. All elements on a page should be able to be traversed by an auxiliary keyboard. As a user uses the keyboard to navigate a page, its focus element should be clear.
6. When a page opens, the first section or element should automatically be placed in focus
7. When a page opens, the title page should be announced

## Accessibility Implementation Guidelines 
**In-App Web View Component**
- Done button in top left
     - Use general done button a11y 
- Refresh button in top right
     - accessibilityLabel= “Refresh”
     - accessibilityHint= “refreshes the page”
     - accessibilityRole= “button”
- Left / right arrows  in bottom left
     - accessibilityLabel= "back" (left) or "forward" (right)
     - accessibilityHint= "navigates to the previous page" (left) OR "navigates to the next page" (right)
     - accessibilityRole= “button”
     - accessibilityState= Denote when enabled or disabled
- Open in browser button in button right
     - accessibilityLabel= “open in browser”
     - accessibilityHint= “opens the page in your device's browser"
     - accessibilityRole= “button”

**Personal & Contact Information Page**
- Back button in top left
     - Use general back button a11y 

**Military Information Page**
- Back button in top left
     - Use general back button a11y 

**Direct Deposit Page**
- Back button in top left
     - Use general back button a11y 

**Edit Direct Deposit Page**
- Cancel button in top left
     - Use general cancel button a11y

**Comment by esther-k** [comment]
Testing...

**Comment by digitalken** [comment]
Was added to TestRail.

**Comment by digitalken** [comment]
PASS 1/14/2021 - Looks good in iOS and Android.  Moving to Design QA.
